src/uagents/protocol.py

```diff
@@ -199,6 +199,12 @@ def manifest(self) -> Dict[str, Any]:
         encoded = json.dumps(manifest, indent=None, sort_keys=True).encode("utf8")
         metadata["digest"] = f"proto:{hashlib.sha256(encoded).digest().hex()}"
```

Comment:
digest is also computed by the `compute_digest` static method.
It takes as input the manifest which will contain the descriptions hence producing a different digest.

Reply:
I modified this method accordingly and also created test for it in test_fields_descr.py, please check it.